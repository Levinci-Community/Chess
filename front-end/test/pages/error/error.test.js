import React from "react";
import { render } from "@testing-library/react";
import ErrorPage from "../../../src/pages/error/index";
import { BrowserRouter as Router } from "react-router-dom";

jest.mock("react-i18next", () => ({
    useTranslation: () => ({
      t: (key) => key,
    }),
  }));
  
  test("renders ErrorPage snapshot", () => {
    const { container } = render(
      <Router>
        <ErrorPage />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });