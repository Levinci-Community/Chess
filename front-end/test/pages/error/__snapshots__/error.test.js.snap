// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ErrorPage snapshot 1`] = `
<div>
  <div
    class="chakra-container css-vl7sen"
  >
    <div
      class="chakra-stack css-1kbzohu"
    >
      <h2
        class="chakra-heading css-zov4ca"
      >
        Oops! Something went wrong.
      </h2>
      <p
        class="chakra-text css-0"
      >
        We're sorry, but there was an error processing your request. Please try again later.
      </p>
    </div>
  </div>
</div>
`;
